coralproject/talk
# [CORL-2554] Show full size tweets in comments

## What does this PR do?

Allow tweets to fill the width of a comment so they don't use mobile styling when on desktop. Scales appropriately for mobile as well.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Go to stream
- Post tweet URL
- Add tweet to comment
- See full width large tweet
 
## How do we deploy this PR?

No special considerations.
